packages/mad-tag-ocr/README.md

```diff
@@ -0,0 +1,53 @@
+# React Native Mad Optical Character Recognition (OCR)
+
+This library adds a camera component that automatically recognizes text in real-time. The text is
+highlighted on the screen with a boundary box and is selectable by clicking the screen. This is
+useful for tasks such as scanning tags on platforms and other facilities.
+
+## 🖌️ How to use
+
+This library exports a component called `OCRCamera` that you can import into your project. The code
+below shows how it can be used.
+
+Remember to call the `requestPermission` function from `react-native-vision-camera` before using the
+OCR camera. Also, if your project is using the [React Navigation](https://reactnavigation.org/)
+package, you can call `useFocusEffect` to make sure permission is requested every time you navigate
+to the OCR camera. This ensures the user is always asked for permissions if permission is not
+already given.
+
+```tsx
+export const MyComponent = () => {
+    const { hasPermission, requestPermission } = useCameraPermission();
+
+    useFocusEffect(() => void requestPermission());
+
+    const onScanResult = (tag: string) => {
+        // do someting here
+    };
+
+    const onScanCancel = () => {
+        // do something here
+    };
+
+    return <OCRCamera onSelectTag={onScanResult} onClose={onScanCancel} />;
+};
+```
+
+Note that the OCR camera only supports viewing in portrait mode.
+
+## ⚙️ Installation
+
+This library has the following peer dependencies. You have to install these in your project.
+
+```
+"@shopify/react-native-skia": "1.2.3",
```

Comment:
Are all of these packages dealt with by Expo?
I know react native skia 1.2.3 is the version that Expo 51 recommends, so any lower Expo versions would complain if a user installed these versions in their project.
If these can be managed automatically by Expo, maybe just refer to the basic installation of each package instead of listing explicit version numbers to ensure we don't have to maintain these versions in the README all the time

Reply:
Seems like Skia is the only of these packages that are listed in the expo documentation https://docs.expo.dev/versions/latest/sdk/skia/

Good idea, I will update the readme